README.md

```diff
@@ -2,43 +2,40 @@
 
 ## Prerequisites
 
-* `nodejs` en `yarn`
+* `nodejs`, `npm`
 
-Use `npm` to install `yarn`:
+## Getting started
 
 ```
-> npm install yarn
+>  Use npm install
+>  Use gulp (to run 'build-sass' and 'views' tasks)
```

Comment:
`gulp`.

This is the "Getting started" section. At this point we're not interested which tasks run, we want to get started